Jenkinsfile

```diff
@@ -1,2 +1,2 @@
 // Build the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin()
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Comment:
Could you update the description to explain / link-to-an-explanation of what this does and why it's a good idea?